app/kumactl/cmd/install/testdata/install-control-plane.cni-experimental-enabled.golden.yaml

```diff
@@ -463,7 +463,7 @@ spec:
               command:
                 - cat
                 - /tmp/ready
-          command: [ "/bin/sh", "-c", "--" ]
+          command: [ "/busybox/sh", "-c", "--" ]
```

Comment:
I'm not super happy that we need to change the path 😢, also distroless does not have `#!/usr/bin/env` which